lib/sherlock_homes/driver.rb

```diff
@@ -0,0 +1,56 @@
+module SherlockHomes
+  class Driver
+
+    class << self
+
+      def phantomjs_options
+        options = [
+          '--ignore-ssl-errors=yes',
+          '--ssl-protocol=any',
+          '--web-security=no',
+          '--load-images=false',
+          '--local-to-remote-url-access=true'
+        ]
+
+        options << [
+          "--proxy=#{SherlockHomes.config.proxy_host}:#{SherlockHomes.config.proxy_port}",
+          '--proxy-type=http'
+        ] if SherlockHomes.config.proxy?
+
+        options
+      end
+
+      def options
+        options = {
+          js_errors: false,
+          debug: false,
+          inspector: false,
+          timeout: SherlockHomes.config.timeout,
+          phantomjs_options: phantomjs_options
+        }
+
+        options.merge!(
+          logger: nil,
+          phantomjs_logger: StringIO.new,
```

Comment:
Could we:
1. make the logger's configurable in a separate pull
2. Make the default logger a Ruby logger that goes to stdout